src/CMakeLists.txt

```diff
@@ -189,6 +189,7 @@ target_sources(can_interface PRIVATE
   CAN/CANMotor_PSOC.cpp
   CAN/CANMotor.cpp
   CAN/CANUtils.cpp
+  CAN/can_motor.cpp
```

Comment:
After you move the can_motor files, can you put add `can_motor.cpp` to the `real_world_interface` target instead?